bootcamp/tutorials/milvus_usages/funnel_search_with_matryoshka.ipynb

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
<p>Instead of using the regular embedding model&nbsp;like <code><a href="https://huggingface.co/sentence-transformers/all-MiniLM-L12-v2" target="_blank">sentence-transformers/all-MiniLM-L12-v2</a></code></p><p> </p><p>"as in the previous example," - there isn't a previous example here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>